actions/workflows/st2_finalize_release.yaml

```diff
@@ -136,6 +136,19 @@ st2cd.st2_finalize_release:
                 branch: master
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - wait_for_community_docker_image
+        wait_for_community_docker_image:
```

Comment:
Additionally, can we move entire Docker stuff to the end of the workflow, after `ship_api_docs`?

In that case, if something goes wrong with Docker it won't block the other important tasks in the workflow.

Reply:
Sure thing.